src/redux/books/BookCard.js

```diff
@@ -0,0 +1,60 @@
+import './BookCard.css';
+
+export default function BookCard() {
+  const books = [
+    {
+      author: 'Robert Kiyosaki',
+      title: 'Rich Dad, Poor Dad',
+      category: 'Finance',
```

Comment:
- [ ] Please move **all** the components under the src/redux folder to a components folder.

Reply:
Updated the files structure.